<?xml version="1.0"?>
<launch>

    <arg name="pset"/>

    <arg name="n"           value="2"/>
    <arg name="enable_log"  value="true"/>

    <rosparam file="$(find wp3_tests)/params/$(arg pset).yml"/>
    <param name="enable_log" value="$(arg enable_log)"/>

    <include file="$(dirname)/$(arg n)Vehs/$(env HOSTNAME).launch"/>

    <node name="Vehicle" pkg="wp3_tests" type="Vehicle.py" output="screen"/>

</launch>

